client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.tsx

```diff
@@ -332,11 +336,9 @@ export function getEnhancedTasks(
 
 							window.location.assign( plansUrl );
 						},
-						badge_text: ! task.completed ? null : translatedPlanName,
-						disabled:
-							( task.completed || ! domainUpsellCompleted ) &&
-							! isBlogOnboardingFlow( flow ) &&
-							! isSiteAssemblerFlow( flow ),
+						badge_text: task.completed ? translatedPlanName : task.badge_text,
+						subtitle: getPlanTaskSubtitle( task ),
```

Comment:
I selected a premium style and didn't upgrade in the Assembler. The Launchpad resulted in:

![Screenshot 2023-12-06 at 10 25 08 PM](https://github.com/Automattic/wp-calypso/assets/797888/78912bfd-7bb8-4626-9dc9-31f64a1f8e85)

`plan_completed` doesn't seem to have `subtitle`.